src/xrandr_extend/main.py

```diff
@@ -88,7 +88,18 @@
     help="Preview command without executing it",
     action="store_true",
 )
-
+parser.add_argument(
+    "--auto-scale",
+    help="Scale external display automatically to match primary dpi",
+    action="store_true"
+)
+parser.add_argument(
+    "--ext-scale",
+    help="Sets the scale factor of the ext. display",
+    nargs=2,
```

Comment:
Why a tuple? Won't it be sufficient to use a float?
```suggestion

```

Reply:
Probably. I only got confused because of the formatting in which xrandr expects it.